gp-includes/formats/format-php.php

```diff
@@ -0,0 +1,189 @@
+<?php
+/**
+ * GlotPress Format PHP class
+ *
+ * @since 4.0.0
+ *
+ * @package GlotPress
+ */
+
+/**
+ * Format class used to support PHP file format.
+ *
+ * @since 4.0.0
+ */
+class GP_Format_PHP extends GP_Format {
+	/**
+	 * Name of file format, used in file format dropdowns.
+	 *
+	 * @since 4.0.0
+	 *
+	 * @var string
+	 */
+	public $name = 'PHP (.php)';
```

Comment:
As the [Performant Translations](https://wordpress.org/plugins/performant-translations/) plugin uses the ".mo.php" extension, what do you think about changing:
```
public $name = 'PHP (.php)';
public $extension = 'php';
```
to
```
public $name = 'PHP (.mo.php)';
public $extension = 'mo.php';
```
because we are going to add the "mo." to each file when we automatically generate them.

Reply:
Sounds good 👍 

Note that the file extension is not 100% defined yet, there was also a suggestion to use `.l10n.php` or similar.

I'll update it once it's settled.

Reply:
Ok. If it is ok to you, we can merge this PR with the current extension and when you will have the final extension, we can update it in a new PR.